internal/internal_workflow_testsuite_test.go

```diff
@@ -2421,11 +2428,18 @@ func (s *WorkflowTestSuiteUnitTest) Test_LocalActivityRetry() {
 		}
 		ctx = WithLocalActivityOptions(ctx, lao)
 
-		var result int32
-		err := ExecuteLocalActivity(ctx, localActivityFn).Get(ctx, &result)
+		err := ExecuteLocalActivity(ctx, nonretriableFn).Get(ctx, nil)
+		badBug, ok := err.(*CustomError)
+		s.True(ok)
+		s.Equal("bad-bug", badBug.Reason())
+
+		var result string
+		err = ExecuteLocalActivity(ctx, retriableFn).Get(ctx, &result)
 		if err != nil {
-			return int32(-1), err
+			return "", err
 		}
+
+		Sleep(ctx, time.Hour)
```

Comment:
what's the purpose of this Sleep?

Reply:
ah, sorry.  leftovers from testing it in a workflow.Go (to make sure that wasn't changing behavior).  I'll remove it.